### ISSUE TITLE
Scrubbing Timeline Crashes Program

### ISSUE THREAD

**Issue by BryanArnold** [NateMccomb/TeslaCamViewerII#13]
When clicking and dragging the playhead on the timeline for more than just a short time, the program crashes instantly.

**Comment by NateMccomb** [comment]
Send me a copy of the recent log file TCViewerLog-2019-xx-xx.txt and the time frame that it last happen to TeslaCamViewer@Gmail.com.

**Comment by NateMccomb** [comment]
Looks like I have found what is crashing it. TCV is not report a crash because it's VLC and the ntdll.dll that crashes from what I found in windows event viewer. I think I was able to address this issue by delaying (250ms) how fast TCV switches between event times while scrolling. VLC is sometimes temperamental when you tell it to do something while it's in the middle of doing the same thing. Another example of this is commanding VLC to stop while the player is already stopped as seen from issue #7. This update can be found in 2019.52.6.0 or newer.
